README.md

```diff
@@ -1,93 +1,283 @@
 
-For federated and incubating projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "Nominate new member of a Project Team" below, the petitioner is granted write access and added to the team.
+#### Tracking Funding
 
-**Steering Council** members are excluded from these votes and retain commit rights to all community projects as stated above. They can add or remove themselves from Project Teams as they see fit in order to indicate which projects they are involved with, have a say in voting for write access petitions, etc. Steering Council members may not add themselves to a Project Team in order to vote on a petition for commit access if they were not already on that team when the petition was made.
+Nominated members will provide a list of organizations that fund above 25% of their time. Nominees will need to provide a complete “chain of funding” for each funding source. For example: “GitHub, a part of Microsoft”, or “CS Dept at Western Michigan University, *Computational Reproducibility Made Easy grant*, NHGRI, NIH, US Federal Government.”
 
+The Steering Council determines the appropriate funding level(s) to consider when looking for common funding with existing and proposed members. The funding source of any Steering Council member can be put to a vote under the policy for [All Other Items](#all-other-items) below.
+
+Council members need to keep their funding documentation up to date, and notify the council whenever it changes. Should a change in funding state create a shared funding situation, the council members should decide who among themselves should move to emeritus status. If no agreement can be reached, the Steering Council will hold a simple vote to determine which member is moved to emeritus status under the [Resolve Overlapping Funding](#resolve-overlapping-funding) voting provision below.
+
+In cases where people have absolutely no funding related to conda, we still document this funding state, but we do not require people to list their irrelevant funding. The "no funding" state can be held by an unlimited number of Steering Council members.
+
+What about cases where someone works on conda for work they are doing, but improving/directing the conda ecosystem is not a specific component of any of their funding? Generally, such cases should be considered as "no funding."
```

Comment:
Am I right to interpret this that employees of a corporation who happen to work on the conda ecosystem but it isn't their direct job function should be considered "no funding"?

If so, is there anything stopping a malicious organization from having a bunch of people indirectly work and end up as "no funding" in order to seize control?

Reply:
My understanding is that if someone pays you to work on conda, direct or indirect, then you are funded. If you only work on conda on unpaid time, then you are not funded. A grad student doing conda in their free time is thus unfunded, but folks at Voltron data working on conda as part of their job are funded.

Reply:
Developers working on something on the weekend may be "not funded" in the same way as a grad student is. Think this is Keith's point.

If that work became important for the day job, that individual would then become funded. They would then be obligated to report their change in status per the governance document.

Since there is a limitation of 2 funded people per company, if there were others that were funded, the company would need to update who they want to keep in the council when this change occurred.

So no I don't think a company could takeover the steering council in this way, but they could mess up on process steps here and there could be consequences for those errors.

Reply:
Right. These things can be put to a vote should something happen.

Reply:
If we're trying to protect against a malicious entity then I think relying on people to report faithfully will not work. They could easily claim they're working on unpaid time and are therefore "not funded".

> My understanding is that if someone pays you to work on conda, direct or indirect, then you are funded. If you only work on conda on unpaid time, then you are not funded. A grad student doing conda in their free time is thus unfunded, but folks at Voltron data working on conda as part of their job are funded.

I understand you'd want to guard against a company like Voltron Data having too much influence, but this puts folks in a bit of a precarious situation. I.E. myself and others are primarily conda users and when we encounter sharp edges we try to help smoothen them out by contributing to the conda ecosystem. Voltron Data as a company doesn't track employee's time so there isn't really a concept of "company time" and because we work on so much OSS we generally know about everything going on with our employees OSS work. Our primary job functions and our paychecks are not based on our conda work, but Voltron Data is definitely happy to encourage and support OSS work.

Additionally, if someone values being on the conda steering committee it actively discourages them from considering companies that will possibly dictate that they need to move to emeritus status to keep their other 2 folks on the steering community, even if the job only leads to indirect contributions to the conda ecosystem.

Reply:
If an entity goes through all of the trouble to convince a bunch of people to lie in an attempt to hijack the project, then I think there would be so many other repercussions that it'd render the conda organization as a group essentially non-functional anyways. 

@CJ-Wright has brought up similar points on employment.

Reply:
FWIW, this vote will pass in ~6 hours depending on time zone which we did specify. 

Happy to revisit but it's pretty late in the hour on this one.

Reply:
This is a good discussion and I appreciate your feedback on this @kkraus14, I look forward to deepening it in the future to make sure that unfunded individuals at companies feel represented in the community.

I want to echo what @jakirkham said about the steering council being a focal point for too many things and too little focus on OTHER ways to contribute, even though the governance policy has many ways to do that. E.g. project teams are perfectly reasonable ways to contribute to the conda ecosystem and are largely self-governed. For irregular contributors these are probably best suited.

The steering council, on the other hand, should really only deal with organizational problems, and administrative work and exists to make sure conda is governed sustainably.

IMHO the focus on the steering council alone (due to Anaconda dominating the project maintenance) was a reason why the governance policy (as designed originally 2 years ago) didn't result in more active participation. As such, the new steering council system that limits funding overreach exists to encourage more companies and organizations to put their money where their mouth is. That there is a part to that that is requiring some level of trust, should not surprise us given the Open Source nature of conda.

Reply:
@beckermr Since it was not explicitly specified I assumed we're using the good old [Anywhere on Earth (AoE) "timezone"](https://en.wikipedia.org/wiki/Anywhere_on_Earth), to be as inclusive as possible. This has worked well in other Open Source communities that I've participated in the past.

Reply:
I agree that this is the start, not the endpoint. Should we have the energy to pursue more governance modifications in the future, I'd be happy to help drive those along as well within reason.

Anywhere on earth is excellent and I will be using that a lot from now on!

Reply:
> The point around recruitment of talent by companies being hindered by this rule is an interesting point (Idk if that particular concern came up)

It's actually the opposite I am more concerned about. If an individual is on the steering council and wishes to continue to be, it potentially hinders their ability to change companies and grow their career as other companies may already have 2 members on the steering council.

Reply:
Yeah sorry Keith I may not have articulated that point well, but my intended meaning was the same as yours.

Reply:
@kkraus14: Yes. :-(

Weighted voting could help address this, while preserving limits on any one organization.

We need to find a cadence for making significant (requiring a vote) changes to governance.  We could reconsider weighting the next time we do this.